src/Flowpipes/setops.jl

```diff
@@ -286,30 +286,6 @@ end
     return !_geq(q, min(X)) || !_leq(q, max(X))
 end
 
-# if X is polyhedral and Y is the set union of half-spaces, X ∩ Y is empty iff
-# X ∩ Hi is empty for each half-space Hi in Y
-# NOTE the algorithm below solves an LP for each X ∩ Hi; however, we can proceed
-# more efficiently using support functions
-# see LazySets.is_intersection_empty_helper_halfspace
-@commutative function isdisjoint(X::AbstractPolytope{N},
```

Comment:
about this one do we have now its corresponding LazySets method which implements similar algorithm?

Reply:
No, but is this important? We can add it if you think it is worth it.

Reply:
i dont much trust the coverage of unit tests in RA; while it has some, i think there are many relevant codepaths that are only tested when we run ARCH-COMP, reachability benchmarks and so on. 

so while removing code is good, i think it would be nice to first make sure we have good integration of models in other packages

Reply:
Hm okay, but it is not clear to me when this point comes. Is there a concrete action to take?
As I said, should we integrate the method in LazySets? I think this method is just a relic from the pre-v2 era.

Reply:
right, the action would be to check that `isdisjoint` between `X` and `Y` here works now (and the implementation is comparatively good to the one here; i see it tries to use a faster method for 2D).

Reply:
Yes, it works:

```julia
julia> using LazySets

julia> X = convert(HPolytope, rand(BallInf))
HPolytope{Float64, ReachabilityBase.Arrays.SingleEntryVector{Float64}}(HalfSpace{Float64, ReachabilityBase.Arrays.SingleEntryVector{Float64}}[HalfSpace{Float64, ReachabilityBase.Arrays.SingleEntryVector{Float64}}([1.0, 0.0], 0.6976622066483278), HalfSpace{Float64, ReachabilityBase.Arrays.SingleEntryVector{Float64}}([0.0, 1.0], 1.934025349777169), HalfSpace{Float64, ReachabilityBase.Arrays.SingleEntryVector{Float64}}([-1.0, 0.0], 1.07029034741529), HalfSpace{Float64, ReachabilityBase.Arrays.SingleEntryVector{Float64}}([0.0, -1.0], -0.16607279571355105)])

julia> Y = UnionSetArray([rand(HalfSpace), rand(HalfSpace)])
UnionSetArray{Float64, HalfSpace{Float64, Vector{Float64}}}(HalfSpace{Float64, Vector{Float64}}[HalfSpace{Float64, Vector{Float64}}([0.6632389695542212, 1.450211466956407], -0.08298385732870821), HalfSpace{Float64, Vector{Float64}}([0.369439555521565, -0.1166531696701808], 1.4408453008536524)])

julia> isdisjoint(X, Y)
false
```